sentry-kotlin-multiplatform/src/androidMain/kotlin/io/sentry/kotlin/multiplatform/extensions/SentryAndroidOptionsExtensions.kt

```diff
@@ -9,4 +10,9 @@ internal fun SentryOptions.toAndroidSentryOptionsCallback(): (SentryAndroidOptio
 
     // Apply Android specific options
     it.isAttachScreenshot = this.attachScreenshot
+    it.sdkVersion = sdk.toJvmSdkVersion()
+    it.sdkVersion?.addPackage(
+        BuildKonfig.SENTRY_ANDROID_PACKAGE_NAME,
+        BuildKonfig.SENTRY_ANDROID_VERSION
+    )
```

Comment:
Can `it.sdkVersion` be null here? or can `sdk.toJvmSdkVersion()` return null?

Reply:
`it.sdkVersion` can be null but `toJvmSdkVersion()` cannot